kiwi_keg/keg.py

```diff
@@ -139,14 +143,19 @@ def main():
             image_version=args['--image-version'],
             track_sources=args['--write-source-info']
         )
-        image_generator = KegGenerator(
-            image_definition=image_definition,
-            dest_dir=args['--dest-dir']
-        )
         if args['--dump-dict']:
+            try:
+                image_definition.populate()
+            except KegError:  # pragma: no cover
+                pass
```

Comment:
dumping the dictionary is not important ?
as in, if it fails we don't care ?

Reply:
`image_definition.populate()` may raise due to the new schema validation, but we still want to be able to dump the dictionary for debugging purposes anyway, so the error is ignored at this point.